app/Http/Controllers/SubmissionsController.php

```diff
@@ -82,4 +82,13 @@ public function store(Request $request)
     return ["status" => 1];
   }
 
+  public function feedback(Request $request)
+  {
+    $submission = Submission::find($request->submission_id);
+    $submission->grade = $request->grade;
+    $submission->feedback = $request->feedback;
+    $submission->save();
+    return json_encode(["status" => 1]);
```

Comment:
I realised you don't need the `json_encode` anymore, so you can just do `return ["status" => 1];`
Sorry my fault